core/account-id/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
 name = "near-account-id"
 version = "0.0.0"
-authors = ["Near Inc <hello@nearprotocol.com>"]
+authors.workspace = true
 publish = true
 # Please update rust-toolchain.toml as well when changing version here:
 rust-version = "1.60.0"
-edition = "2021"
+edition.workspace = true
```

Comment:
Yeah, here I'd probably avoid sharing edtition as well. When we switch to 2024, we won't immediatlly want to switch account id, as it's a proper crates.io library with backward compatability guarantees